### PR TITLE
otpilot: Clean-up output from main.

### PR DESCRIPTION
**Pull request by osenft**

### CODE REVIEW

File: userspace/otpilot/src/main.rs [hunk 1]
```diff
@@ -169,10 +161,11 @@ fn run() -> TockResult<()> {
 
     //////////////////////////////////////////////////////////////////////////////
 
-    let mut flash_read: [u8; flash::MAX_BUFFER_LENGTH] = [0; flash::MAX_BUFFER_LENGTH];
-    writeln!(console, "reading flash ...")?;
-    flash::get().read(0x5000, &mut flash_read, 8)?;
-    writeln!(console, "flash_read = {:?}", &flash_read[0..8])?;
+    {
+        let mut flash_read: [u8; flash::MAX_BUFFER_LENGTH] = [0; flash::MAX_BUFFER_LENGTH];
```
Comment (dann21) [inline userspace/otpilot/src/main.rs]: Somebody is likely to object to the "redundant" type specifier.

Not me.

:-)

File: userspace/otpilot/src/main.rs [hunk 2]
```diff
@@ -169,10 +161,11 @@ fn run() -> TockResult<()> {
...
+        flash::get().read(0x5000, &mut flash_read, 8)?;
```
Comment (dann21) [inline userspace/otpilot/src/main.rs]: Is `0x5000` a magic number?  Given that this is inside a "demo" function, do we care?

File: userspace/otpilot/src/main.rs [hunk 3]
```diff
@@ -195,11 +188,11 @@ fn run() -> TockResult<()> {
                 Ok(()) => {}
                 Err(why) => {
                     // Ignore error from writeln. There's nothing we can do here anyway.
-                    let _ = writeln!(console, "Device: Error processing SPI packet: {:?}", why);
+                    let _ = writeln!(console, "SPI processor: Error {:?}", why);
                     if spi_device::get().is_busy_set() {
                         if let Err(_) = spi_device::get().end_transaction_with_status(true, false) {
                             // Ignore error from writeln. There's nothing we can do here anyway.
-                            let _ = writeln!(console, "Device: Error ending transaction.");
+                            let _ = writeln!(console, "SPI device: end_transaction error.");
```
Comment (dann21) [inline userspace/otpilot/src/main.rs:187]: Why not print the error?

(Guessing it is content free, except insofar as it informs us that something didn't go right.)

File: userspace/otpilot/src/main.rs [hunk 4]
```diff
@@ -213,7 +206,7 @@ fn run() -> TockResult<()> {
                 Ok(()) => {}
                 Err(_) => {
                     // Ignore error from writeln. There's nothing we can do here anyway.
-                    let _ = writeln!(console, "Device: Error processing console event.");
+                    let _ = writeln!(console, "Console processor: Error.");
```
Comment (dann21) [inline userspace/otpilot/src/main.rs:201]: Why not print the error?

File: userspace/otpilot/src/main.rs [hunk 5]
```diff
@@ -224,7 +217,7 @@ fn run() -> TockResult<()> {
                 Ok(()) => {}
                 Err(_) => {
                     // Ignore error from writeln. There's nothing we can do here anyway.
-                    let _ = writeln!(console, "Device: Error processing GPIO event.");
+                    let _ = writeln!(console, "GPIO processor (event): Error.");
```
Comment (dann21) [inline userspace/otpilot/src/main.rs:212]: Why not print the error?

File: userspace/otpilot/src/main.rs [hunk 6]
```diff
@@ -234,7 +227,7 @@ fn run() -> TockResult<()> {
                 Ok(()) => {}
                 Err(_) => {
                     // Ignore error from writeln. There's nothing we can do here anyway.
-                    let _ = writeln!(console, "Alarm: Error processing event.");
+                    let _ = writeln!(console, "GPIO processor (alarm): Error.");
```
Comment (dann21) [inline userspace/otpilot/src/main.rs:222]: Why not print the error?

File: userspace/otpilot/src/main.rs [hunk 7]
```diff
@@ -259,9 +252,9 @@ async fn main() -> TockResult<()> {
     writeln!(console, "clock_frequency: {}", alarm::get().get_clock_frequency())?;
     let result = run();
     if result.is_ok() {
-        writeln!(console, "Returned OK.")?;
+        writeln!(console, "main: returning OK.")?;
     } else {
-        writeln!(console, "Returned error.")?;
+        writeln!(console, "main: returning error.")?;
```
Comment (dann21) [inline userspace/otpilot/src/main.rs:249]: same thing here -- is there no value to printing the error that's being returned?

**Review by dann21 — APPROVED** [review]
LGTM, modulo a bunch of optional nits to pick.

File: userspace/otpilot/src/main.rs (hunk 3 above)
Comment (osenft) [reply userspace/otpilot/src/main.rs:187]: We don't have a printer for TockResult right now. This is covered under #227.

File: userspace/otpilot/src/main.rs (hunk 4 above)
Comment (osenft) [reply userspace/otpilot/src/main.rs:201]: Covered by #227.

File: userspace/otpilot/src/main.rs (hunk 5 above)
Comment (osenft) [reply userspace/otpilot/src/main.rs:212]: Covered by #227.

File: userspace/otpilot/src/main.rs (hunk 6 above)
Comment (osenft) [reply userspace/otpilot/src/main.rs:222]: Covered by #227.

File: userspace/otpilot/src/main.rs (hunk 7 above)
Comment (osenft) [reply userspace/otpilot/src/main.rs:249]: Same as above - we don't have a printer for TockResult and it's covered by #227.

File: userspace/otpilot/src/main.rs (hunk 1 above)
Comment (osenft) [reply userspace/otpilot/src/main.rs]: I removed this demo. It doesn't make any sense anymore and we don't want information from the flash to leak.

File: userspace/otpilot/src/main.rs (hunk 3 above)
Comment (dann21) [reply userspace/otpilot/src/main.rs:187]: :ack:

File: userspace/otpilot/src/main.rs (hunk 4 above)
Comment (dann21) [reply userspace/otpilot/src/main.rs:201]: :ack:

File: userspace/otpilot/src/main.rs (hunk 5 above)
Comment (dann21) [reply userspace/otpilot/src/main.rs:212]: :ack:

File: userspace/otpilot/src/main.rs (hunk 6 above)
Comment (dann21) [reply userspace/otpilot/src/main.rs:222]: :ack:

File: userspace/otpilot/src/main.rs (hunk 7 above)
Comment (dann21) [reply userspace/otpilot/src/main.rs:249]: :ack:

File: userspace/otpilot/src/main.rs (hunk 1 above)
Comment (dann21) [reply userspace/otpilot/src/main.rs]: :ack:

File: userspace/otpilot/src/main.rs (hunk 2 above)
Comment (dann21) [reply userspace/otpilot/src/main.rs]: Given that this function is now gone, never mind.

**Pull request merged**